seattle_lidar/seattle_lidar.ipynb

```diff
@@ -282,17 +286,17 @@
    "source": [
     "def plot(data, **kwargs):\n",
     "    \"\"\"Plot point elevation data, rasterizing by mean elevation\"\"\"\n",
-    "    points = hv.Points(data, kdims=['meterswest', 'metersnorth'], vdims=['Z'])\n",
     "    image_opts = opts.Image(tools=['hover'], cmap='blues_r', colorbar=True,\n",
     "                            width=800, height=800, xaxis=None, yaxis=None)\n",
-    "    return rasterize(points, aggregator=rd.mean('Z'), precompute=True, **kwargs).options(image_opts)"
```

Comment:
Here's what I found in the docs: https://holoviews.org/user_guide/Large_Data.html#cache-initial-processing-with-precompute-true

It appears hvPlot settings also allow it so I've added it as well.

Reply:
I found out a few things digging on this. This example was previously in the EarthML repo, and before that in the datashader repo. This is when it was moved to EarthML (https://github.com/holoviz-topics/EarthML/pull/85), and when `precompute=True` was added. I asked Jean-Luc and he told me the best way to check its effect was to test with and without it :) I tried and saw no difference.

It looks like it confirms this comment from Philipp (who added `precompute` to HoloViews`) https://github.com/holoviz/holoviews/issues/1909#issuecomment-378376733 with:

> `precompute` can be used to cache the last set of data, for dataframes/dask tabular datasets and xarray gridded datasets this makes no difference, for TriMesh/QuadMesh this will make a huge difference

The datasets used on this example are Pandas and Dask dataframes, setting `precompute` should make no difference (except using more memory...).

So I've removed `precompute=True`!